### ISSUE TITLE
Low circulating thrombopoietin concentration

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10244]
**Preferred term label:**
Low circulating thrombopoietin concentration

**Synonyms**

**Definition (free text, please give PubMed ID)**
Opposite of https://hpo.jax.org/app/browse/term/HP:0034600

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0003117

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://omim.org/entry/620478

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000021
- label: Decreased circulating thrombopoietin concentration
- parents: HP:0003117
- pmid: PMID:32150607
- def: Concentration of thrombopoietin in the blood circulation below the lower limit of normal. Thrombopoietin is a glycoprotein hormone produced by the liver and kidney that stimulates the production and differentiation of megakaryocytes.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10244
- synonyms:
